### PR TITLE
fixed cs inverse signal

### PR DESCRIPTION
**Pull request by PSVM-J**
Was inverse signal cs
@Igor89 @acosinwork

### CODE REVIEW

**Comment by Igor89** [comment]
LGTM

File: modules/@amperka/quaddisplay2.js [hunk 1]
```diff
@@ -17,10 +17,12 @@ var QuadDisplay = function(opts) {
     SPI2.setup({mosi: B15, miso: B14, sck: B13});
     this._spi = SPI2;
     this._cs = opts;
+	this._cs.mode('output');
   } else {
     opts = opts || {};
     this._spi = opts.spi;
     this._cs = opts.cs;
+	this._cs.mode('output');
```
Comment (nkrkv) [inline modules/@amperka/quaddisplay2.js]: Linter won’t be happy: mixed tabs and spaces.

File: modules/@amperka/quaddisplay2.js [hunk 2]
```diff
@@ -87,6 +89,7 @@ QuadDisplay.prototype.frame = function(shift) {
     this._shift = this._data.length;
   }
   this._spi.send(this._data.slice(this._shift, this._shift + 4), this._cs);
+  digitalWrite(this._cs,0);
```
Comment (nkrkv) [inline modules/@amperka/quaddisplay2.js]: `digitalWrite(this._cs,1);` has lost above. Is it OK?

Comment (PSVM-J) [reply modules/@amperka/quaddisplay2.js]: Yes, It's work without `digitalWrite(this._cs,1); `

**Pull request merged**